src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
```

Comment:
The getRandomBall method should not be static according to the checklist. Static methods are generally discouraged. Please create an instance of the Lottery class where you need to use this method.